library/src/main/java/com/yayandroid/locationmanager/providers/locationprovider/DispatcherLocationSource.java

```diff
@@ -44,7 +44,7 @@ boolean isGoogleApiErrorUserResolvable(int gpServicesAvailability) {
     }
 
     @Nullable Dialog getGoogleApiErrorDialog(Activity activity, int gpServicesAvailability, int requestCode,
-          OnCancelListener onCancelListener) {
+                                             OnCancelListener onCancelListener) {
```

Comment:
Can you please clean up these spaces since there is no actual change here this file should not be in this PR?